charts/hedera-json-rpc-relay/values.yaml

```diff
@@ -250,15 +255,20 @@ readinessProbe:
   initialDelaySeconds: 20
   timeoutSeconds: 5
 
-# Disabled newman temporarily
 test:
   enabled: false
+  # Note: hoppscotch/hopp-cli does not exist as a standalone Docker image.
+  # The CLI is distributed only as an npm package (@hoppscotch/cli).
+  # We use the official hoppscotch/hoppscotch AIO image which includes Node.js,
+  # and install the CLI at runtime via npm.
   image:
     pullPolicy: IfNotPresent
-    repository: postman/newman
-    tag: 5.3.1-alpine
-  # Custom postman.json in base64 encoding
-  postman: ''
+    repository: hoppscotch/hoppscotch
```

Comment:
Why do we need this base image when on our package json we only need the cli?
As we want to avoid unncesary dependencies, shouldnt we pick a clean baseOS  image pinned to a version with. npm, and instal the cli with a specific version?